src/core/entities/MRTextFieldEntity.js

```diff
@@ -73,7 +107,7 @@ export class MRTextFieldEntity extends MRTextInputEntity {
 
         // Ensure the cursor position is updated to reflect the current caret position
         setTimeout(() => {
```

Comment:
what's this for?

Reply:
if we end up needing to pass the event info/etc to the function within that, we want the timeout to have a delay s.t. the event actually holds all the proper information (since it's a press/trigger type one) otherwise, the events can get delayed so the next time you grab the info, you're grabbing the previous version until the next event gets called (it's like the stack gets delayed by one or more steps if that makes sense)

---

tldr atm this setup is no longer needed since i dont pass the 'event' object to the cursor function anymore, but if we ever do it again, it will be needed with something > 0

^ given this, lmk if you'd want it removed or kept just in case

Reply:
I'd say if we're not using it we can lose it for the sake of cleanliness